test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -73,10 +77,22 @@ source "${DIR}/deploy-kubeflow.sh"
 # Install Argo CLI and test-runner service account
 source "${DIR}/install-argo.sh"
 
+IMAGE_BUILDER_ARG=""
+# When project is not ml-pipeline-test, VMs need permission to fetch some images in gcr.io/ml-pipeline-test.
```

Comment:
remove the comment?

Reply:
The tests are too slow and flaky. Since I already got all the tests passing. I don't want to change this and rerun them. I have removed this in the next PR.